clang/test/Driver/aix-ld.c

```diff
@@ -1120,3 +1120,23 @@
 // RUN:        -c \
 // RUN:   | FileCheck --check-prefixes=CHECK-K-UNUSED %s
 // CHECK-K-UNUSED: clang: warning: -K: 'linker' input unused [-Wunused-command-line-argument]
+
+
+// This test verifies that the linker doesn't include '-latomic' when no sanitizers are enabled 
+// Running this test on non-AIX host will result in an unrelated error 
+// FIXME: Sanitizer interface functions must be exported by export files on AIX
```

Comment:
why do we have this FIXME here? Do you expect separate tests in the future for the FIXME or does the FIXME apply specifically to the test below?